src/ui/components/CardList/CardList.tsx

```diff
@@ -109,6 +109,7 @@ const CardList = <T extends object = object>({
   return (
     <IonList
       lines={lines}
+      data-testid="card-list"
```

Comment:
I didn't realize you created a new component for this. it's not necessary, you can use `IdentifiersList` (or use that component and refactor it).

Reply:
Because this component can be use to credential tab then I think I should create new components that adapt with both credential and identifier tab.